frontend/app/guide-modern-v2/page.tsx

```
@@ -0,0 +1,845 @@
+  Hotel,
+  Music,
+  Sun,
+  Cloud,
+  CloudRain,
+  Info,
+  Download,
+  Share2,
+  Settings,
+  Heart,
+  Ticket,
+  Map,
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)